std/conv.d

```diff
@@ -1732,7 +1735,7 @@ private T toImpl(T, S)(S value)
 /// ditto
 private T toImpl(T, S)(S value, uint radix)
     if (isInputRange!S && isSomeChar!(ElementEncodingType!S) &&
-        !isExactSomeString!T && is(typeof(parse!T(value, radix))))
+        isIntegral!T && is(typeof(parse!T(value, radix))))
```

Comment:
What's with the template constraint change? We can't have that in a doc PR. Please remove it.